Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Applications/Browser/ChromeApplication.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Aquality.Selenium.Core.Applications;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using System;
+
+namespace Aquality.Selenium.Core.Tests.Applications.Browser
+{
+    public class ChromeApplication : IApplication
+    {
+        private TimeSpan implicitWait;
+
+        public ChromeApplication()
+        {
+            Driver = new ChromeDriver();
```

Comment:
is it work if we do not have chromedriver.exe in the PATH?

Reply:
of course not. it is a simple implementation for tests, it's not in the core lib and doesn't go anywhere else, just to test.